servers/jax-rs-tests/src/test/java/org/projectnessie/jaxrs/tests/TestJerseyResteasyH2.java

```diff
@@ -22,4 +22,4 @@
 
 @NessieDbAdapterName(H2DatabaseAdapterFactory.NAME)
 @NessieExternalDatabase(H2TestConnectionProviderSource.class)
-class TestJerseyResteasyH2 extends AbstractTestJerseyResteasy {}
+class TestJerseyResteasyH2 extends AbstractTestDatabaseAdapterResteasy {}
```

Comment:
why is the `Jersey`-part still relevant for this concrete subclass but no longer for the base class?
in this file itself there seems to be nothing Jersey-specific